app/src/main/res/values/colors.xml

```diff
@@ -9,6 +9,7 @@
     <color name="colorPrimary">#008085</color>
     <color name="widechatStatusBar">#0E7075</color>
     <color name="widechat_fab">#00C1B7</color>
+    <color name="widechat_login_button">#08364A</color>
     <!-- End Widchat Colors -->
     <color name="colorAccent">#FF1D74F5</color>
 
```

Comment:
- Some clean up that could happen here I think. If we're in principle not breaking non-widechat we shouldn't be just coopting their colorPrimary should we? 
- We should probably restore their "Main colors" section as they have it. And maybe make our section "Widechat main colors" in case we do similar things further down, like eventually creating a "Widechat text colors".
- The two snake case ones should be camel case for consistency.
- typo: "Widchat"

Reply:
colorPrimary proved to be a difficult one to replace.  The current RC code is not itself factored properly to use this variable in an easily swappable way.  It's being called all over the place, as opposed to a single place, for example in 'styles'.  

I decided that it was going to be easier to maintain by just replacing the color in this one place instead of changing the variable being called in dozens of files. 

Happy to discuss alternatives if you want though....

I can possibly factor this more clearly by commenting out their whole block and replacing it with ours....  and I can fix the typo and snake case stuff....